src/dal/Jalasoft.TeamUp.Resumes.DAL/InvitationsInMemoryRepository.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Jalasoft.TeamUp.Resumes.DAL
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+
+    public class InvitationsInMemoryRepository : IInvitationsInMemoryRepository
+    {
+        private static readonly Invitation[] Invitations = new Invitation[]
+            {
+                new Invitation
+                        {
+                            Id = "5a7939fd-59de-44bd-a092-f5d8434584de",
+                            ProjectId = "724d912-59de-44bd-a092-f5d8434584de",
+                            ProjectName = "Sony",
+                            ResumeId = Guid.Parse("fb14f3f4-7d2e-4c4e-9c50-9997c83a77c9"),
+                            ResumeName = "Jose Ecos",
+                            PictureResume = "photo.png",
+                            TextInvitation = "We invite you to collaborate with the development team",
+                            StartDate = DateTime.Now.AddDays(-10),
+                            ExpireDate = DateTime.Now,
+                            Status = "invited"
+                        },
```

Comment:
https://dev.azure.com/funjaladev/DevLevel3/_wiki/wikis/Project/42/API-resource-for-invitations?anchor=**get-%2F-find-resume%27s-invitations**

the json design indicates another structure, please could you double check, or am I missing something?
@alejandro-castellon-jalafund